src/lib/proxy/create-context.test.ts

```diff
@@ -55,13 +55,34 @@ test('will not blow up if properties is an array', () => {
         properties: ['some'],
     });
 
-    // console.log(context);
-
     expect(context.userId).toBe('123');
     expect(context).not.toHaveProperty('tenantId');
     expect(context).not.toHaveProperty('region');
 });
 
+test('will respect environment set in context', () => {
+    const context = createContext({
+        userId: '123',
+        tenantId: 'some-tenant',
+        environment: 'development',
+        region: 'eu',
+        properties: ['some'],
+    });
+
+    expect(context.environment).toBe('development');
+});
+
+test('will not set environment to be development if not set in context', () => {
+    const context = createContext({
+        userId: '123',
+        tenantId: 'some-tenant',
+        region: 'eu',
+        properties: ['some'],
+    });
+
+    expect(context.environment).toBe(undefined);
```

Comment:
why will it be `undefined` in this test when we sets it to `req.user.envrionment`?

Reply:
sorry, you are testing the "create-context" here. I thought this was an API test for some reason..

Reply:
...meaning: this test is not relevant for the changes code lines.